src/domain/Popovers/DropdownMenu/subcomponents/Section.tsx

```diff
@@ -101,6 +110,27 @@ class Section extends React.PureComponent<ISectionProps, never> {
     }
 
     const Component = this.component
+    if (tooltipMessage) {
+      return (
+        <StyledSection {...this.styleProps}>
+          <Component
+            {...componentProps}
+            onClick={onClick}
+            href={href}
+          >
+            <StyledSectionContent>
+              <Margin margins={{ right: Variables.Spacing.sXSmall }}>
+                {leftComponent && <span className='left-component'>{leftComponent}</span>}
+                {text}
+                {rightComponent && <span className='right-component'>{rightComponent}</span>}
+              </Margin>
+              <FontAwesomeIcon icon='info-circle' type='solid' color={Variables.Color.n400}/>
+            </StyledSectionContent>
```

Comment:
Instead of having two returns, you can conditionally render this part based on `tooltipMessage` like `tooltipMessage ? (JSX) : (JSX)`